.travis.yml

```diff
@@ -13,6 +13,7 @@ script:
 - ./www/bin/phpcs --standard=symfony2 www/src
 - ls -l www/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php
 - ./www/bin/phpunit -c www/app/
+- php ./scripts/coverage-checker.php www/app/logs/clover.xml 24
```

Comment:
Aktuell würde es dann fehlschlagen, denn die Coverage ist unter 24

    $ php coverage-checker.php ../www/app/logs/clover.xml  24
    Code coverage is 23.59%, which is below the accepted 24%